lightly/api/download.py

```diff
@@ -164,46 +129,3 @@ def job(**kwargs):
                 future.result()
             except Exception as ex:
                 warnings.warn(f"Could not download {filename} from {url}")
-
-
-def download_prediction_file(
-    url: str,
-    session: requests.Session = None,
-    request_kwargs: Optional[Dict] = None,
-) -> Dict:
-    """Same as download_json_file. Keep this for backwards compatability.
-
-    See download_json_file.
-
-    """
-    return download_json_file(url, session=session, request_kwargs=request_kwargs)
-
-
-def download_json_file(
```

Comment:
Accidentally removed from LightlySSL, as it is still used in the worker. Now added in the worker in https://github.com/lightly-ai/lightly-core/pull/3938